ts/Core/Defaults.ts

```diff
@@ -2797,6 +2797,22 @@ function setOptions(
     options: DeepPartial<DefaultOptions>
 ): Options {
 
+    if (defaultOptions.defaultCircularOptions && options.xAxis) {
+        merge(
+            true,
+            defaultOptions.defaultCircularOptions,
+            options.xAxis
+        );
+    }
+
+    if (defaultOptions.defaultRadialGaugeOptions && options.yAxis) {
+        merge(
+            true,
+            defaultOptions.defaultRadialGaugeOptions,
+            options.yAxis
+        );
+    }
+
```

Comment:
We should consider an event here, fired on Highcharts, so that we can listen in `RadialAxis.ts` for the `setOptions` event on the `H` object and apply these options.